pkg/build/gobuild.go

```diff
@@ -888,16 +888,15 @@ func (g *gobuild) Build(ctx context.Context, s string) (Result, error) {
 		return nil, err
 	}
 
-	// Take the digest of the base index or image, to annotate images we'll build later.
-	baseDigest, err := base.Digest()
-	if err != nil {
-		return nil, err
-	}
-
 	// Annotate the base image we pass to the build function with
 	// annotations indicating the digest (and possibly tag) of the
 	// base image.  This will be inherited by the image produced.
-	if mt != types.DockerManifestList && !g.preserveMediaType {
+	if !(mt == types.DockerManifestList && g.preserveMediaType) {
```

Comment:
This will add annotations to docker images when preserveMediaType is true, do you want that?

Reply:
Yeah. Docker images (not manifest lists) support annotations, at least as far as GCR lets me push those.